frontend/src/forms/subforms/EvaluationForm.tsx

```diff
@@ -112,6 +113,26 @@ export const filterEmptyFinancials = (evaluations: IFinancial[]) =>
     return !!evaluation.value || (evaluation.key === EvaluationKeys.Appraised && !!evaluation.date);
   });
 
+/**
+ * Get the paginated page numbers that contain errors.
+ */
+const getPageErrors = (errors: any, nameSpace: any) => {
+  const evaluationErrors = getIn(errors, nameSpace);
+  const errorsPerPage = Object.keys(keyTypes).length * NUMBER_OF_EVALUATIONS_PER_PAGE;
+  return _.uniq(
+    _.reduce(
+      evaluationErrors,
+      (acc: number[], error, index) => {
+        if (error) {
+          acc.push(Math.ceil((parseInt(index) + 1) / errorsPerPage));
```

Comment:
because there are multiple instances of possible errors per page, we need to map all errors to the page that they are displayed on, so that user's know which paginated pages contain errors.

Reply:
Is it clear to the user if the error is on a different page than the one they're viewing?

Reply:
The error message calls out the specific page number(s) that have errors.